### PR TITLE
Bugfix/pp9 10846   content portal   update url with the state values

### PR DESCRIPTION
**Pull request by tc4rdpp**

### CODE REVIEW

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      }
+    } else {
+      query[key] = value;
+    }
+  });
+
+  router.navigate([], { queryParams: query });
+}
+
+export function getState(searchInfo: {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts:33]: Rename to getSearchState

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts [hunk 2]
```diff
@@ -0,0 +1,52 @@
+import { Params, Router } from '@angular/router';
+import { SearchInputState } from '../facade/search.facade';
+import { SearchMode } from '../models/search-mode';
+import { AggregationFilter } from '../services/api-services';
+
+export function updateUrlFromState(searchState: SearchInputState, query: Params, router: Router) {
```
Comment (neotek) [inline src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts:6]: Rename to updateUrlFromSearchState.

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts (hunk 1 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts]: Done

File: src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts (hunk 2 above)
Comment (tc4rdpp) [reply src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular/src/lib/utilities/search.util.ts]: Done

**Pull request merged**